kahypar/partition/configuration.h

```diff
@@ -46,6 +46,14 @@ struct PreprocessingParameters {
   bool remove_always_cut_hes = false;
   bool remove_parallel_hes = false;
   MinHashSparsifierParameters min_hash_sparsifier = MinHashSparsifierParameters();
+  
+  //Louvain-Configuration
+  bool use_louvain = false;
+  bool use_louvain_in_ip = true;
```

Comment:
This default configuration seems to be inconsistent. Why should use_louvain_in_ip be true even if use_louvain is false?